android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -163,7 +163,12 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
   protected @Nullable String mDownloadingMessage = null;
   protected @Nullable String mLackPermissionToDownloadMessage = null;
 
+  private static Set<String> originWhitelist;
+
   public RNCWebViewManager() {
+    originWhitelist = new HashSet<>();
+    originWhitelist.add("https://alchemy.veriff.com/");
```

Comment:
capture: discuss ways to have this inputed from the client

Reply:
can we add at the same place that we use the origin whitelist for the allowable domains